eng/common/scripts/Cadl-Project-Sync.ps1

```diff
@@ -1,127 +0,0 @@
-# For details see https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/Cadl-Project-Scripts.md
```

Comment:
You should not be deleting files under `eng\common` directly in a language repo. In fact I don't know how this didn't fail as we have a CI check for this. I've filed a bug to investigate this.

Reply:
I got this back in sync with https://github.com/Azure/azure-sdk-tools/issues/5882